### PR TITLE
logger: log datetime when on debug mode

### LINKED ISSUE #3282: loggers: include timestamp to see how long operation takes
In logger output (mostly `dvc command -v`) we need to add time to see operation differences, see how long it took overall, etc. It's very valuable information.

### PR DESCRIPTION
**Pull request by skshetry**
* [x] ❗ Have you followed the guidelines in the [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core) list?

* [x] 📖 Check this box if this PR **does not** require [documentation](https://dvc.org/doc) updates, or if it does **and** you have created a separate PR in [dvc.org](https://github.com/iterative/dvc.org) with such updates (or at least opened an issue about it in that repo). Please link below to your PR (or issue) in the [dvc.org](https://github.com/iterative/dvc.org) repo.

* [x] ❌ Have you checked DeepSource, CodeClimate, and other sanity checks below? We consider their findings recommendatory and don't expect everything to be addressed. Please review them carefully and fix those that actually improve code or fix bugs.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

Fixes #3282

Screenshot from 2020-02-14 17-14-23

### CODE REVIEW

File: dvc/logger.py [hunk 1]
```diff
@@ -59,12 +59,28 @@ class ColorFormatter(logging.Formatter):
...
+        # also, skip INFO as it is used for UI
+        if (
+            self._current_level() != logging.DEBUG
+            or record.levelno == logging.INFO
+        ):
+            return ""
+
+        return "{color}{date}{nc} ".format(
+            color=colorama.Fore.GREEN,
+            date=super().formatTime(record, datefmt),
```
Comment (skshetry) [inline dvc/logger.py]: The default output looks like `2020-02-14 17:13:44,627`. I did not really tried fighting with the format and went with default one. Plus, this supports setting `datefmt` from dictConfig.

Comment (skshetry) [reply dvc/logger.py]: If we need a different format later on, we can just change in the `dictConfig` anyway. :slightly_smiling_face:

Comment (efiop) [reply dvc/logger.py]: Yeah, a bit long for my taste. We could shorten stuff like year (cut it off or shorten to "20"), but it won't make a big deal. I'm ok with leaving it as is.

File: tests/unit/test_logger.py [hunk 2]
```diff
@@ -4,25 +4,40 @@
...
     "blue": colorama.Fore.BLUE,
+    "green": colorama.Fore.GREEN,
     "red": colorama.Fore.RED,
     "yellow": colorama.Fore.YELLOW,
     "nc": colorama.Fore.RESET,
 }
 
 
+@pytest.fixture()
+def dt(mocker):
```
Comment (pared) [inline tests/unit/test_logger.py]: :+1:

**Pull request merged**